.github/workflows/terraform.yml

```diff
@@ -47,9 +47,12 @@ jobs:
           KEYCLOAK_V2_DEV_URL=https://dev.sandbox.loginproxy.gov.bc.ca
           KEYCLOAK_V2_TEST_URL=https://test.sandbox.loginproxy.gov.bc.ca
           KEYCLOAK_V2_PROD_URL=https://sandbox.loginproxy.gov.bc.ca
-          KEYCLOAK_V2_DEV_CLIENT_SECRET=${{ secrets.DEV_KEYCLOAK_V2_DEV_CLIENT_SECRET }}
-          KEYCLOAK_V2_TEST_CLIENT_SECRET=${{ secrets.DEV_KEYCLOAK_V2_TEST_CLIENT_SECRET }}
-          KEYCLOAK_V2_PROD_CLIENT_SECRET=${{ secrets.DEV_KEYCLOAK_V2_PROD_CLIENT_SECRET }}
+          KEYCLOAK_V2_DEV_USERNAME=${{ secrets.PROD_KEYCLOAK_V2_DEV_USERNAME }}
```

Comment:
Are these supposed to be the other way around, down here would be the DEV_KEYCLOAK_V2 vars and below would be the PROD ones?

Reply:
Haha yes side effects of copy pasting, good catch